Source/GSStoryboardTransform.m

```diff
@@ -715,13 +773,52 @@ - (NSMapTable *) processConnections: (NSArray *)connectionsArray
               {
                 ident = [[NSUUID UUID] UUIDString];
               }
+            attr = [obj attributeForName: @"popoverAnchorView"];
+            NSString *av = [attr stringValue];
+            attr = [obj attributeForName: @"popoverBehavior"];
+            NSString *pb = [attr stringValue];
+            NSPopoverBehavior behavior = NSPopoverBehaviorApplicationDefined; 
```

Comment:
Shouldn't this mapping be done when unarchiving?

Reply:
It can't be.  I am constructing the segue during the process of creating the proxy.  It is never directly decoded.

Reply:
Yes, but these values get decoded and there they could be mapped.